protocols/imap/pom.xml

```diff
@@ -122,6 +122,12 @@
             <artifactId>apache-james-mailbox-store</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
The TestId class may be extracted to another poject.
It's strange to depend on a mailbox project in protocols.